src/ghastoolkit/octokit/github.py

```diff
@@ -99,6 +99,53 @@ def getPullRequestCommits(self) -> list[str]:
                 result.append(commit.get("sha"))
         return result
 
+    def getPullRequestComments(self) -> list[dict[str, int | str]]:
```

Comment:
```suggestion
    def getPullRequestComments(self) -> list[dict[str, Union[int, str]]]:
```

To support Python v3.9 you will need to use `Union` type

Reply:
Missed that one, thanks!

Reply:
Might need to import the type too

```python
from typing import Optional, Tuple, Union
```